src/cloud-cli/userdb.ts

```diff
@@ -0,0 +1,105 @@
+import axios from "axios";
+import { createGlobalLogger } from "../telemetry/logs";
+import { getCloudCredentials } from "./utils";
+
+export async function createUserDb(host: string, port: string, dbName: string, adminName: string, adminPassword: string, sync: boolean) {
+  const logger = createGlobalLogger();
+  const userCredentials = getCloudCredentials();
+  const bearerToken = "Bearer " + userCredentials.token;
+  
+  try {
+    await axios.post(`http://${host}:${port}/${userCredentials.userName}/databases/userdb`, 
+    {"Name": dbName,"AdminName": adminName, "AdminPassword": adminPassword},
+    {
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: bearerToken,
+      },
+    });
+
+    logger.info(`Successfully started creating database: ${dbName}`);
+    let status = ""
```

Comment:
If we are not using `status` elsewhere, we might want to place it in the `if(sync)` branch. Also we might want to set it to `Creating` - or whatever is the default AWS status to represent this state -- by default?